edb/repl/__init__.py

```diff
@@ -850,6 +864,10 @@ def _command_errverbose(
             print('> ' + '\n> '.join(srv_tb.strip().split('\n')))
             print()
 
+    def _command_semicolon_hint(self, arg: Optional[str]) -> None:
+        if arg and arg.strip()[-1] == ';':
```

Comment:
This can be an IndexError unless you do it this way (I suggested this exact code before):

```suggestion
        if arg.strip() and arg.strip()[-1] == ';':
```